doc/man7/EVP_RAND-TEST_RAND.pod

```diff
@@ -0,0 +1,88 @@
+=pod
+
+=head1 NAME
+
+EVP_RAND-TEST_RAND - The test EVP_RAND implementation
+
+=head1 DESCRIPTION
+
+A test random number generator which should never be used for production.
```

Comment:
hmm the term production might be misleading - we may use this internally in self tests

Reply:
Decent point.  Reworded.